intake/gui/source/gui.py

```diff
@@ -62,7 +62,7 @@ def __init__(self, cats=None, sources=None, done_callback=None, **kwargs):
         self.description = Description(source=self.sources)
 
         self.plot = DefinedPlots(source=self.sources,
-                                 visible=self.plot_widget.value,
+                                 visible=False,
```

Comment:
I don't think this is needed.

Reply:
The value is always False at this point (it's defined just above), thought it good to avoid the obfuscation

Reply:
I think it is good to make sure it is always the same. It'd be weird if you set it to True here.

Reply:
reverted